status-panels-lib/status-panels/panels.libsonnet

```
@@ -0,0 +1,95 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+
+local stat = g.panel.stat;
+local row = g.panel.row;
+function(
+  title,
+  showIntegrationVersion,
+  integrationVersion,
+  statusPanelsTarget,
+  panelsHeight,
+  panelsWidth,
+  rowPositionY,
+)
+
+  {
+    local this = self,
+
+    integrationStatusInit(targets, title='Integration Status')::
```

```suggestion
    integrationStatusInit(targets, title='Integration status')::
```
as in [grafana style guide](https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/#sentence-case-capitalization)

---

Fixed